llpc/tool/amdllpc.cpp

```diff
@@ -1611,9 +1612,33 @@ int main(int argc, char *argv[]) {
   if (isFailure())
     return onFailure();
 
-  // The first input file is a pipeline file or LLVM IR file. Assume they all are, and compile each one
-  // separately but in the same context.
-  if (isPipelineInfoFile(expandedInputFiles[0]) || isLlvmIrFile(expandedInputFiles[0])) {
+  // Build relocatable shader cache. This requires all inputs to be .pipe files.
```

Comment:
We should mention that this is still a work in progress, and eventually we will want to accept plain spir-v files.

Reply:
Done